torchtext/experimental/models/roberta_model.py

```diff
@@ -0,0 +1,74 @@
+import os
+import torch
+import torch.nn as nn
+from torchtext.experimental.modules import TransformerEncoder
+from .roberta_transform import RobertaTransform
+from torchtext.utils import download_from_url, extract_archive, files_exist, load_args_from_json
+
+
+class RobertaModel(nn.Module):
+    """Contain a Roberta encoder model."""
+
+    def __init__(self, args, encoder):
```

Comment:
In general I think it gives a bad user experience to construct models via opaque `args` or `config`.
It makes it much harder to just jump in an interpreter and explore / experiment with the model. It is much simpler if the arguments are just passed in right away.

If we want to have a method for constructing the model via configs, we could have a builder function like `build_model` which takes the config and unpacks it, while still leaving the `__init__` take the unpacked arguments